src/targets/axis_2400.ini

```diff
@@ -16,7 +16,7 @@ monitor_speed = 420000
 src_filter = ${env_common_esp32.src_filter} -<rx_*.cpp>
 lib_deps =
     ${env_common_esp32.lib_deps}
-    bodmer/TFT_eSPI
+    https://github.com/Bodmer/TFT_eSPI#40cd5bfe7b62bfc1606df930e0f0d540d8f20618
```

Comment:
Should this be `bodmer/TFT_eSPI @ 2.4.01`

Reply:
There is no release with that tag so it won't work.  But the version number has been updated in the commit hash used.

Reply:
Sure there is https://platformio.org/lib/show/1559/TFT_eSPI/installation
But sadly it doesn't parse with platformio 😢 , So I'm going to raise another ticket!

Reply:
So a new version has been released by the developer with proper versioning. `2.4.11`. So we should change the dependency to `bodmer/TFT_eSPI @ 2.4.11`